contracts/src/v0.8/llo-feeds/Verifier.sol

```diff
@@ -316,13 +316,79 @@ contract Verifier is IVerifier, ConfirmedOwner, TypeAndVersionInterface {
     bytes memory offchainConfig,
     Common.AddressAndWeight[] memory recipientAddressesAndWeights
   ) external override checkConfigValid(signers.length, f) onlyOwner {
+    _setConfig(
```

Comment:
Under what circumstances would we use setConfig as opposed to setConfigFromSource?

Reply:
IMO `setConfig` is a convenience method for setting the config on the source `Verifier`, and `setConfigFromSource` can be used on the dependent `Verifier`s to set their config. Any alternatives you want to consider?